### ISSUE TITLE
[EN][Dungeon][npc]Shadowfang Keep hc: Dread Scryer

### ISSUE THREAD

**Issue by stanq87** [Atlantiss/BugTracker#9887]
**Description**: the mob Dread Scryer stads still and only atacks from melle when a player is in range

**Current behaviour**: read Description

**Expected behaviour**: Dread Scryer should be capable of moving, activly try to come to melle range with a player/tank, and should use their abilities

http://www.wowhead.com/npc=47141/dread-scryer#abilities:mode=heroic

https://youtu.be/O0PTPxDQh6k?t=500

**Steps to reproduce the problem**:

1.  go to hadowfang Keep hc, the last room before last boss
2. 
3. 

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat)

**Comment by TranceAccount** [comment]
According to the Legacy Smart AI, he should start moving when he reaches 15% mana...

**Comment by stanq87** [comment]
well maybe the issue is that they dont seem to have mana ?

**Comment by TranceAccount** [comment]
Yes I saw that when I was looking into this (wrong unit_class in template), but the time before this mob reaches 15% mana is like after 2-4 minutes, and most of the time they are already dead in a dungeon like situation.

**Comment by stanq87** [comment]
ok ok them not moving is not so important, them not casting is
